andela-iakande/document-management-system
# Feature/#144440357/setup client environment

#### What does this PR do?
This PR set up client environment for the document manager application

#### Description of Task to be completed?
install react
configure webpack
set up babel
build a landing page including sign in, sign up and dashboard pages


#### How should this be manually tested?
1) clone the repo
2) run npm start to start the app and you will see the landing page

#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
setup client environment

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A
